app/user/rest.py

```diff
@@ -133,6 +130,14 @@ def update_user_attribute(user_id):
     return jsonify(data=user_to_update.serialize()), 200
 
 
+def set_reply_to(recipient, template):
```

Comment:
I would suggest adding some unit tests to this method please. The `test_post_user_attribute_with_updated_by_sends_notification_to_international_from_number` test is good but definitely doesn't cover all the logic of this method